sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -5,12 +5,14 @@
 ```ts
 
 import { AmqpMessage } from '@azure/core-amqp';
+import { ConnectionConfig } from '@azure/core-amqp';
 import { DataTransformer } from '@azure/core-amqp';
 import { delay } from '@azure/core-amqp';
 import { Delivery } from 'rhea-promise';
 import Long from 'long';
 import { MessagingError } from '@azure/core-amqp';
 import { RetryOptions } from '@azure/core-amqp';
+import { SharedKeyCredential } from '@azure/core-amqp';
```

Comment:
Can you run the build and ensure that the above changes to the file are indeed needed anymore?

Reply:
updated